runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/BpmnModeler/BpmnModeler.types.ts

```diff
@@ -37,3 +37,253 @@ export interface ModelerHTMLCanvasElement extends HTMLCanvasElement {
     zoom: (precition: string, scale: string) => void;
     viewbox: (opts?: ModelerViewboxOpts) => ModelerViewboxResult;
 }
+
+export enum ModelerEvents {
```

Comment:
bruh, that's impressive

Reply:
![MicrosoftTeams-image (6)](https://user-images.githubusercontent.com/62908149/220588678-acc65d8f-2420-49e9-9fca-23783f9e7644.png)